### PR TITLE
Nc news 5

### PR DESCRIPTION
**Pull request by PGallagher93**

### CODE REVIEW

**Comment by Harrison365** [comment]
Nice work. 
You should only need to group by articles.article_id in your SQL but as long as your tests are passing, the rest is great.

File: __tests__/app.test.js [hunk 1]
```diff
@@ -81,6 +81,30 @@ describe("GET:/api/articles/:article_id", () => {
   });
 });
 
+describe.only("GET 200: /api/articles", () =>{
+  test("returns the correct array of article objects with a comment count and ordered by date desc", () =>{
+
+    return request(app).get("/api/articles").expect(200).then(({body}) =>{
+      const articles = body
+      expect(articles).toHaveLength(13)
+      expect(articles).toBeSortedBy("created_at")
```
Comment (johnhutcheon) [inline __tests__/app.test.js]: We also want to check that the articles are in descending order 👍

File: models/articles-models.js [hunk 2]
```diff
@@ -11,4 +11,16 @@ exports.findArticleById = (id) => {
...
-};
\ No newline at end of file
+};
+
+exports.readArticles = () => {
+    return db.query(`SELECT articles.author, title, articles.article_id, topic, articles.created_at, articles.votes, article_img_url, COUNT(comments.article_id) AS comment_count FROM articles
+    LEFT JOIN comments
+    ON articles.article_id = comments.article_id
+    GROUP BY articles.author, title, articles.article_id, topic, articles.created_at, articles.votes, article_img_url
+    ORDER BY articles.created_at;`).then(({rows}) =>{
```
Comment (johnhutcheon) [inline models/articles-models.js]: You'll just need to add one more word in this query to get the descending order 😄

**Review by johnhutcheon — CHANGES_REQUESTED** [review]
Hey Phil, just see my comments and make the small amendments please. Thanks and well done! 🙂

**Pull request merged**